classloader-leak-prevention/classloader-leak-prevention-core/src/test/java/se/jiderhamn/classloader/leak/prevention/cleanup/MoxyCleanUpTest.java

```diff
@@ -17,7 +17,13 @@ protected void triggerLeak() throws Exception {
     // Class.forName("org.eclipse.persistence.jaxb.compiler.Property", true, leakSafeCL);
     Class.forName("org.eclipse.persistence.jaxb.compiler.CompilerHelper", true, leakSafeCL);
     
-    CompilerHelper.getXmlBindingsModelContext();
+    try {
+      CompilerHelper.getXmlBindingsModelContext();
+    }
+    catch(IllegalAccessError e) {
+      // CompilerHelper.getXmlBindingsModelContext() tries to load some internal classes that 
```

Comment:
Will the test still pass on Java 11?

Reply:
They do! This is similar to the JceSecurityCleanupTest. I can add [the same comment](https://github.com/mjiderhamn/classloader-leak-prevention/blob/24a75571ba4f191556075065af48ffe95b73b48f/classloader-leak-prevention/classloader-leak-prevention-core/src/test/java/se/jiderhamn/classloader/leak/prevention/cleanup/JceSecurityCleanUpTest.java#L23) if this makes things more clear.

You can not tell that from the travis build, since it errors, because the SAAJEnvelopeFactoryParserPoolCleanUpTest cannot be compiled. I have a branch containing all changes, including the changed SAAJEnvelopeFactoryParserPoolCleanUpTest, so the build no longer errors. On that branch all tests that I have worked on [succeed on Travis.](https://github.com/Nikos410/classloader-leak-prevention/runs/34384014)